admin/typos

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Checks for known typos in all files known by git, with limited exceptions.
+#
+# To auto-correct all detected typos:
+# $ ./admin/typos --write-changes
+#
+# To show other options:
+# $ ./admin/typos --help
+
+# XXX TODO TYPOS SHOULD CATH SORRY CATCH MY KNOWN TYPOS) HERE
+# Can use some of the same options from typos-cli, for exmapl sorry example sorry example:
+# XXX XXX typos seems to miss this one =============>  ^^^^^^ (VS Code Spell Check DOES seem to catch sorry ccatch sorry catch this one)
+# XXX XXX dang typos seems to mis sorry miss this one as well: ===========================================>> ^^^^^^
+
+set -e
+
+# XXX TODO FIXUP SPELLING - SORRY TYPO - see if typos cathces these
+# XXX XXX typos MISSES MY misspelling of ENV VARIABLE here:
+EXCLUD_FILE_PATTERNS=$(echo \
```

Comment:
reported typo of "exclude" here: https://github.com/crate-ci/typos/issues/1221#issuecomment-2648925186